docs/guides/private-docker.md

```diff
@@ -0,0 +1,51 @@
+# Private docker registry
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+Until [#1459](https://github.com/cortexlabs/cortex/issues/1459) is addressed, you can use a private docker registry for your Predictor images by following this guide.
```

Comment:
We should mention that this guide is only necessary using images from a private dockerhub registry when deploying to the cluster. Or maybe have a section for local and cluster here.

Reply:
Good call! I went with a note, but could also make a section if you think it's better. We can discuss live